project/src/components/Form.js

```diff
@@ -43,50 +43,39 @@ const myRouterComponent = withRouter (class Form extends Component {
         }
     }
 
-    onChange= (e, field, validMethod, name)=>{
+    onChange(e, field, validMethod, name){
         let val = e.target.value;
-        let valid = validMethod(field.value);
+        let valid=(validMethod===true)?true: validMethod(field.value);
```

Comment:
Please, change this line. This is not the best way to check the function or returned value.